services/src/main/java/org/keycloak/protocol/oidc/endpoints/UserInfoEndpoint.java

```diff
@@ -25,14 +25,17 @@
 import org.keycloak.common.ClientConnection;
 import org.keycloak.common.VerificationException;
 import org.keycloak.common.constants.ServiceAccountConstants;
-import org.keycloak.crypto.SignatureProvider;
-import org.keycloak.crypto.SignatureSignerContext;
-import org.keycloak.crypto.SignatureVerifierContext;
+import org.keycloak.crypto.*;
```

Comment:
By following the keycloak's convention, it might be better not to include wildcard "*" in an import directive.

Reply:
Hello @tnorimat, thanks for your feedback.
My IDE (IntelliJ IDEA) seems to include wildcard automatically when imported class for a package is >=  5, I see there are other wildcard in the project with same convention (e.g org.keycloak.models.* in org.keycloak.models.cache.infinispan.RealmAdapter, java.util.* in org.keycloak.storage.ldap.idm.query.internal.LDAPQuery).
Let me know if I need to provide a fix anyway or if you have further considerations.